qutebrowser/javascript/webelem.js

```diff
@@ -55,7 +55,7 @@ window._qutebrowser.webelem = (function() {
         try {
             caret_position = elem.selectionStart;
         } catch (err) {
-            if (err instanceof DOMException &&
+            if (err.constructor.name === "DOMException" &&
```

Comment:
I'm not sure why, but this instanceof seems to be failing for elements in the iframe. I'm not sure why...

Reply:
That's because you check against the `DOMException` of the top window, but another `DOMException` is created and used for the frame window.

```js
window.frames[0].document.getElementsByTagName("p")[0] instanceof HTMLParagraphElement
// false
window.frames[0].document.getElementsByTagName("p")[0] instanceof window.frames[0].HTMLParagraphElement
// true
```